docs/source/deploy.rst

```diff
@@ -15,6 +15,9 @@ You can use Hey_ to check if your proxy is behaving properly.
 
 An `example configuration`_ file for fast clients with Nginx_:
 
+*nginx.conf*
```

Comment:
It would be better to use the `caption` option of `literalinclude`:

```rst
 .. literalinclude:: ../../examples/nginx.conf
     :language: nginx
     :caption: nginx.conf
```